x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -991,18 +991,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           before(async () => {
             await setCustomDashboardsEnabled(true);
             await loginWithReadOnlyUser();
-            await pageObjects.common.navigateToApp(HOSTS_VIEW_PATH);
-            await pageObjects.header.waitUntilLoadingHasFinished();
-
-            await pageObjects.timePicker.setAbsoluteRange(
-              START_SYNTHTRACE_DATE.format(DATE_PICKER_FORMAT),
-              END_SYNTHTRACE_DATE.format(DATE_PICKER_FORMAT)
-            );
+            // Setting the params directly will help us reduce flakiness (and clicking the wrong tab)
+            // as we already test the flow (in #Single Host Flyout) here we can directly navigate to test the permissions
+            const dashboardsTabSearchParams = `?_a=(dateRange:(from:%27${DATE_WITH_HOSTS_DATA_FROM}%27,to:%27${DATE_WITH_HOSTS_DATA_TO}%27),filters:!(),limit:100,panelFilters:!(),query:(language:kuery,query:%27%27))&controlPanels=(cloud.provider:(fieldName:cloud.provider,grow:!f,order:1,title:%27Cloud%20Provider%27,type:optionsListControl,width:medium),host.os.name:(fieldName:host.os.name,grow:!f,order:0,title:%27Operating%20System%27,type:optionsListControl,width:medium),service.name:(fieldName:service.name,grow:!f,order:2,title:%27Service%20Name%27,type:optionsListControl,width:medium))&tableProperties=(detailsItemId:host-1-Linux,pagination:(pageIndex:0,pageSize:10),sorting:(direction:desc,field:alertsCount))&assetDetails=(dateRange:(from:%27${DATE_WITH_HOSTS_DATA_FROM}%27,to:%27${DATE_WITH_HOSTS_DATA_TO}%27),name:host-1,tabId:dashboards)`;
```

Comment:
Do we need the `controlPanel` object?

Reply:
I tried to remove some params but the problem is that we need them for the hosts view to load properly - we need to load it as this tab is inside the flyout (I was getting the error page and decided to leave them all to avoid unexpected errors) 
I will still try again and comment here the result 👍

Reply:
It worked locally with removing only the `controlPanel` (the `panelFilters` and the other props in `_a` need to be there still), Removed [here](https://github.com/elastic/kibana/pull/210988/commits/52fe98f6bac9333f4e0e182134011e358ab1a5f2) - I will rerun the flaky test runner just in case